C++/3. Operators.md

```diff
@@ -2,42 +2,38 @@
 
 ## Start
 
-There are certain operators which do addition, and things like that in C++. In this file, I will teach you the basic operators:
+C++ provides certain operators which do addition, and things like that in C++. In this file, I will teach you the basic arithmetic operators:
 
 - `+`
 - `-`
 - `*`
 - `/`
-- `%`
 
 ## What they do
 
-The `+` operator adds things together. \
-The `-` operator subtracts two things. \
-The `*` operator multiplies things together. \
-The `/` operator divides two things. \
-The `%` operator find the reminder of a division.
+The `+` operator adds two numbers. \
+The `-` operator subtracts two numbers. \
+The `*` operator multiplies two numbers. \
+The `/` operator divides two numbers.
 
 ### Code
 
 ```cpp
 #include <iostream>
 
 int main() {
-  int three = 3;
-  int four = 4;
+  double three = 3.0;
+  double four = 4.0;
 
-  int addition = three + four;
-  int subtraction = four - three;
-  int multiplication = three * four;
-  int division = four / three;
-  int remainder = four % three;
```

Comment:
again we need this. Also change the var name cause it is wrong (i see blue highlighting)

Reply:
What name do you suggest instead?

Reply:
eh anything. Maybe remainderOfNumbers